core/src/main/java/org/elasticsearch/common/network/InetAddresses.java

```diff
@@ -23,10 +25,12 @@
 import java.nio.ByteBuffer;
 import java.util.Arrays;
 import java.util.Locale;
+import java.util.regex.Pattern;
 
 public class InetAddresses {
     private static int IPV4_PART_COUNT = 4;
     private static int IPV6_PART_COUNT = 8;
+    private static final Pattern MASK_PATTERN = Pattern.compile("[\\.|/]");
```

Comment:
I think this should be an actual pattern for a ipv4 address? Otherwise the code below would allow passing eg `0/0/0/0/0` and parse it as an ip?